backend/lib/security.js

```diff
@@ -248,7 +248,12 @@ function authenticate () {
     }
     const bearer = decrypt(encryptedBearer)
     assert.ok(bearer, 'The decrypted bearer token must not be empty')
-    user.auth = { bearer }
+    const auth = user.auth = { bearer }
+    if (typeof createClient === 'function') {
+      Object.defineProperty(user, 'api', {
```

Comment:
add enumerable: false property to make it clearer why you use definedProperty here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#Description

Reply:
✔️